tedana/workflows/tedana.py

```diff
@@ -420,15 +422,15 @@ def tedana_workflow(data, tes, out_dir='.', mask=None,
     elif ctab is not None:
         raise IOError('Argument "ctab" must be an existing file.')
 
-    if isinstance(manacc, str):
-        manacc = [int(comp) for comp in manacc.split(',')]
-
     if ctab and not mixm:
         LGR.warning('Argument "ctab" requires argument "mixm".')
         ctab = None
     elif manacc is not None and not mixm:
         LGR.warning('Argument "manacc" requires argument "mixm".')
         manacc = None
+    elif manacc is not None:
+        # coerce to list of integers
+        manacc = [int(m) for m in manacc]
```

Comment:
Should we add a `LGR.info` message here to note that the components were manually accepted ?

Reply:
That info is logged later, during the actual selection:
https://github.com/ME-ICA/tedana/blob/75bdda8bbc0e1025fdd07fe59e8d8a8b4d754771/tedana/selection/tedica.py#L34-L37

Reply:
Hmm. I couldn't find it in the integration test logs. I'll check again !

Reply:
OK, so the five echo test isn't showing the logs from re-running. The logs only show the first pass:

```
INFO     tedana.selection.tedica:tedica.py:117 Performing ICA component selection with Kundu decision tree v2.5
INFO     REPORT:tedica.py:118 Next, component selection was performed to identify BOLD (TE-dependent), non-BOLD (TE-independent), and uncertain (low-variance) components using the Kundu decision tree (v2.5; Kundu et al., 2013).
INFO     REFERENCES:tedica.py:122 Kundu, P., Brenowitz, N. D., Voon, V., Worbe, Y., Vértes, P. E., Inati, S. J., ... & Bullmore, E. T. (2013). Integrated strategy for improving functional connectivity mapping using multiecho fMRI. Proceedings of the National Academy of Sciences, 110(40), 16187-16192.
INFO     REPORT:tedana.py:587 Rejected components' time series were then orthogonalized with respect to accepted components' time series.
INFO     tedana.io:io.py:226 Writing optimally combined time series: /tmp/data/five-echo/TED.five-echo/ts_OC.nii.gz
```

I'm wondering if it's because the second call is using `_main()` instead of `tedana_workflow()` [here](https://github.com/ME-ICA/tedana/pull/598/files#diff-04bba601abd1f8097e716bfd1800302cR115)

Reply:
@emdupre I think that's why, unfortunately.

Reply:
Opened in #606.